services/spar/src/Spar/Scim/User.hs

```diff
@@ -160,6 +161,24 @@ instance Scim.UserDB ST.SparTag Spar where
       )
       $ deleteScimUser tokeninfo uid
 
+authIdFromUser :: TeamId -> User -> Maybe AuthId
+authIdFromUser stiTeam user = do
```

Comment:
No, I don't think this function should exist.  But maybe we need to make another change to reading old users back from the database, if a scim-only user is created without `sso_id`?  Let's talk!